jobs/tendrl-jobs-build-triggers.yaml

```diff
@@ -0,0 +1,77 @@
+- job-template:
+    name: "tendrl-build-{package_name}"
+    display-name: 'Tendrl-{package_name}'
+    description: 'Do not edit this job through the web!'
+    project-type: freestyle
+    defaults: global
+    disabled: false
+    concurrent: false
+    quiet-period: 0
+    block-downstream: false
+    block-upstream: false
+    node: 'master'
+    triggers:
+        - timed: 'H H(0-7) * * *'
+        - pollscm:
+            cron: 'H/10 * * * *'
+            ignore-post-commit-hooks: false
+    scm:
+        - git:
+            url: 'https://github.com/Tendrl/{package_name}'
+            branches:
+                - "*/master"
+            skip-tag: true
+            wipe-workspace: true
+    builders:
+        - shell: |
+            #!/bin/bash -xe
+            RELEASE=$(date +"%m_%d_%Y_%H_%M_%S")
+            sed -i tendrl-{package_name}.spec -e "s/^Release: .*/Release: $RELEASE/"
```

Comment:
It would be great to use `make snapshot` here instead of manipulating the .spec file with `sed`